owncloudApp/src/debug/res/values/setup.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- App name  and other strings-->
+    <string name="app_name" formatted="false">ownCloud</string>
```

Comment:
debug values??? Should these values be here?

Reply:
Well, these values were put here to not show the wizard (`<bool name="wizard_enabled">false</bool>`) when executing the tests, otherwise will fail. But there's maybe other way to achieve this, I can check whether we are in debug mode and disable the wizard dynamically.